.github/workflows/tact.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       matrix:
         node-version: [18.x]
         os: [ubuntu-latest, windows-latest, macos-latest]
-    
```

Comment:
let's remove these unrelated whitespace changes, we'll fix this eventually and add a check in CI so it does not happen again, but manual unrelated changes are discouraged

Reply:
Thanks for noticing, the IDE was making in on every save. Fixed